src/test/lrsql/ops/query/generate_test.clj

```diff
@@ -0,0 +1,72 @@
+(ns lrsql.ops.query.generate-test
+  (:require [clojure.test :refer [deftest testing is]]
+            [lrsql.ops.query.generate :as generate]))
+
+(def fake-cond-map
+  {"completed_a" {"id"     "6fbd600f-d17c-4c74-801a-2ec2e53231f7"
```

Comment:
The outermost keys in the cond-map returned from SQL will be kehywords, not strings. The rest is correct though.

Reply:
changed